libnmstate/nm/sriov.py

```diff
@@ -129,7 +129,7 @@ def _remove_sriov_vfs_in_setting(vfs_config, sriov_setting, vf_ids_to_remove):
 
 def _has_sriov_capability(ifname):
     dev = device.get_device_by_name(ifname)
-    if nmclient.NM.DeviceCapabilities.SRIOV & dev.props.capabilities:
+    if dev and (nmclient.NM.DeviceCapabilities.SRIOV & dev.props.capabilities):
```

Comment:
nit: You can just return the condition result.

Reply:
Hey. You are not supposed to be working.
Thanks.

Reply:
You are right. Just trying to keep my brain alive :)